README.md

```diff
@@ -1133,6 +1133,7 @@ A [human resources management system](https://en.wikipedia.org/wiki/Human_resour
 - [FHEM](https://fhem.de/fhem.html) - FHEM is used to automate common tasks in the household like switching lamps and heating. It can also be used to log events like temperature or power consumption. You can control it via web or smartphone frontends, telnet or TCP/IP directly. ([Source Code](https://svn.fhem.de/trac)) `GPL-3.0` `Perl`
 - [Gladys](https://gladysassistant.com/) - Gladys is a privacy-first, open-source home assistant. ([Source Code](https://github.com/GladysAssistant/Gladys)) `Apache-2.0` `Nodejs`
 - [Home Assistant](https://home-assistant.io/) - Open-source home automation platform. ([Demo](https://home-assistant.io/demo/), [Source Code](https://github.com/home-assistant/core)) `Apache-2.0` `Python`
+- [ioBroker](https://www.iobroker.net/) - ioBroker is an integration platform for the Internet of Things, focused on Building Automation, Smart Metering, Ambient Assisted Living, Process Automation, Visualization and Data Logging. ([Source Code](https://github.com/ioBroker/ioBroker)) `MIT` `Nodejs`
```

Comment:
Congratulations on the contribution!


> Keep the short description under 250 characters and use [sentence case](https://en.wikipedia.org/wiki/Letter_case#Sentence_case)
for it, even if the project's webpage or readme uses another capitalisation
such as title case, all caps, small caps or all lowercase.


Please change your description to match scentence case.

Reply:
Please start the sentence with `Integration platform ...`

Reply:
I changed it. Shall I also change `Internet of Things` or are you fine with that?

Reply:
Id leave it as its commonly referred to like that.